### PR TITLE
Added some best stock examples to the cookbook, changed some people

### PR DESCRIPTION
**Pull request by Lnaden**

### CODE REVIEW

File: docs/yamlpages/cookbook.rst [hunk 1]
```diff
@@ -7,6 +7,169 @@ Having all the options laid out in front of you is good for advanced users, but
...
 A series of examples that may help you understanding how to put together these options found on the other pages.
 
+----
+
+.. _yaml_ex_implicit:
+
+Example: Implicit Solvent Binding Simulation:
+=============================================
+
+This example sets up para-xylene binding to T4-Lysozyme in implciit solvent.
```
Comment (andrrizzi) [inline docs/yamlpages/cookbook.rst]: Typo implciit

File: docs/yamlpages/cookbook.rst [hunk 2]
```diff
@@ -7,6 +7,169 @@ Having all the options laid out in front of you is good for advanced users, but
...
 
+----
+
+.. _yaml_ex_implicit:
+
+Example: Implicit Solvent Binding Simulation:
+=============================================
+
+This example sets up para-xylene binding to T4-Lysozyme in implciit solvent. 
+We reccommend these settings as a good baseline simulation setup.
```
Comment (andrrizzi) [inline docs/yamlpages/cookbook.rst]: Typo reccommend

File: docs/yamlpages/cookbook.rst [hunk 3]
```diff
@@ -7,6 +7,169 @@ Having all the options laid out in front of you is good for advanced users, but
...
+Although you do not have to set all these options (e.g. the ligand may be a pdb file, not just a SMILES string), many 
+of these options are good stock settings to remember.
+
+In this example:
+
+* Targeting folders for input files
+* Configuring Output files
+* Setting good stock options for implicit simulations
+* Setting up simple protein/ligand binding simulation
+* NVT ensemble
```
Comment (andrrizzi) [inline docs/yamlpages/cookbook.rst]: Default pressure value is `1 * atmosphere`, not `null` so I think this is still NPT.

File: docs/yamlpages/cookbook.rst [hunk 4]
```diff
@@ -7,6 +7,169 @@ Having all the options laid out in front of you is good for advanced users, but
...
+----
+
+.. _yaml_ex_explicit:
+
+Example: Absolute Binding free energy in explicit solvent
+=========================================================
+
+This example takes the same para-xylene binding to T4-Lysozyme system as before, but now uses an explicit solvent setup, 
+minimal options, and automatic water addition (TIP3P).
+This example also sets up MPI so you can run on a cluster; assumes 4 nodes are available
```
Comment (andrrizzi) [inline docs/yamlpages/cookbook.rst]: The YAML script does not have control on whether yank is run through `mpirun` or simply through `python`. It only has automatic detection on whether it's running with MPI or not.

File: docs/yamlpages/cookbook.rst [hunk 5]
```diff
@@ -7,6 +7,169 @@ Having all the options laid out in front of you is good for advanced users, but
...
+* Setting good stock options for explicit simulations
+* Configuring MPI
+* NPT ensemble
+
+.. code-block:: yaml
+
+   options:
+     minimize: yes
+     verbose: yes
+     mpi: yes
```
Comment (andrrizzi) [inline docs/yamlpages/cookbook.rst]: This is option is not currently supported. If it's still in some example we should remove it.

Comment (jchodera) [inline docs/yamlpages/cookbook.rst]: Which option? `mpi`?

Comment (andrrizzi) [inline docs/yamlpages/cookbook.rst]: Yep! Yank automatically detects whether it's running with MPI or not.

**Comment by jchodera** [comment]
Can you explicitly use the term "best practices" in here somewhere on the recommended protocol page, maybe with reference to the material on alchemistry.org?

**Comment by jchodera** [comment]
Thanks, looks good to me! Merge when ready!

**Pull request merged**